### DISCUSSION TITLE
how do I connect two xmpp accounts?

### DISCUSSION THREAD

**Discussion by mstyp** [SpectrumIM/spectrum2#416] [Q&A]
hello. I am really struggling with the documentation, and what I need to do to connect two xmpp accounts (one running localy via yunohost, one running on dismail.de). It would be really helpful if there was a sample config file out there that lets you bridge / connect / two xmpp accounts.

I think I installed spectrum2 successfully, and I think I cant start the service (server? idk) after I specified the config file with `spectrum2 -c /etc/spectrum2/transports/spectrum.cfg` and by then running `spectrum2_manager start` (this took me several hours to figure out, you guys *really* need to update your documentation)

But I just copied the built in config file, and I cant figure out how to make the config file bridge the two xmpp services. I was hoping I would be able to pair them like how I pair two callendars with vdirsyncer.

```
[pair bob_calendar]
a = "bob_calendar1"
b = "bob_calendar2"

# Calendars also have a color property
metadata = ["displayname", "color"]

[storage bob_calendar1]
type = "caldav"
url = "https://owncloud.example.com/remote.php/caldav/"
password = "password"

[storage bob_calendar2]
type = "caldav"
url = "https://nextcloud.example.com/remote.php/caldav/"
password = "password"
```

The closest I can find is https://spectrum.im/documentation/tutorials/beginners_guide.html but that is not my use case, and I cant figure how to make it match my use case. I can see how to adapt it to connect to my yunohost xmpp server, but then what do I add to the file to get it to connect the second one?

Looking at the documentation for the config files are even more befuddling. none of the lines listed in https://spectrum.im/documentation/configuration/config_file.html are in the sample config files, for example.

**Comment by vitalyster** [comment]
I have the same confused feeling while reading your problem description :D
It is totally unclear what do you want to achieve. Please, elaborate on the problem you want to solve using spectrum, not on your confusion with documentation. Maybe spectrum is a  totally unrelated tool?

**Reply by mstyp** [reply]
Nothing quite like the feeling of mutual confusion. 

Essentially, Instead of connecting an IRC account and an XMPP account, I want to connect 2 XMPP accounts on different servers. So messages sent to an xmpp account acount1@dismail.de to appear as messages in another xmpp account on a yunohost server, account2@yunohost.local.
